server/WFCORE-6815-reload-domain-mode-to-stability-level.adoc

```diff
@@ -0,0 +1,161 @@
+---
+categories:
+  - core
+  - management
+# Add any category for this proposal
+# if missing, add it to _data/widfly-categories and use its id
+---
+= [Community] Ability to reload a domain to a different stability level
+:author:            Yeray Borges
+:email:             yborgess@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WFCORE-6728 introduces the ability to reload a standalone server to a different stability level. This is mainly for the testsuite to be able to reload the server to a different stability level before running the tests, and to reload back to the original stability level after the tests have completed.
+
+This proposal is to extend this functionality to domain mode and provide a method to reload the domain controller to a different stability level for testing purposes.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE[WFCORE-6815]
+
+=== Related Issues
+
+//* https://issues.redhat.com/browse/WFLY[WFLY-XXXX]
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [x] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:brian.stansberry@redhat.com[Brian Stansberry]
+* mailto:kkhan@redhat.com[Kabir Khan]
+* mailto:jperkins@redhat.com[James Perkins]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+WildFly Core. The main changes will be in the controller, host-controller and testsuite/shared modules.
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [X] Managed domain
+
+* [] OpenShift s2i
+
+* [] Bootable jar
```

Comment:
Very minor, these should be `[ ]` so they show up as checkboxes. Same with the first one too... ...again minor :)

Reply:
Done, thanks for reviewing this @jamezp